I'm sorry, but an uncaught exception occurred.

While running game code:
  File "game/init.rpy", line 78, in script
    python:
  File "game/init.rpy", line 81, in <module>
    aubrey_sprite = aubrey_necklace
NameError: name 'aubrey_necklace' is not defined

-- Full Traceback ------------------------------------------------------------

Full traceback:
  File "game/init.rpy", line 78, in script
    python:
  File "C:\Users\Nicholas\Downloads\renpy-8.2.3-sdk.7z(1)\renpy-8.2.3-sdk\renpy\ast.py", line 823, in execute
    renpy.python.py_exec_bytecode(self.code.bytecode, self.hide, store=self.store)
  File "C:\Users\Nicholas\Downloads\renpy-8.2.3-sdk.7z(1)\renpy-8.2.3-sdk\renpy\python.py", line 1178, in py_exec_bytecode
    exec(bytecode, globals, locals)
  File "game/init.rpy", line 81, in <module>
    aubrey_sprite = aubrey_necklace
NameError: name 'aubrey_necklace' is not defined

Windows-10-10.0.19045 AMD64
Ren'Py 8.2.3.24061702
Reunion {size=20}1.0{/size}
Thu Aug 15 01:11:23 2024
